language_server/src/lsp/feature_workspace_folders.cpp

```diff
@@ -120,4 +132,28 @@ void feature_workspace_folders::did_change_watched_files(const json&, const json
         paths.begin(), paths.end(), std::back_inserter(c_uris), [](const std::string& s) { return s.c_str(); });
     ws_mngr_.did_change_watched_files(c_uris.data(), c_uris.size());
 }
+
+void feature_workspace_folders::send_configuration_request()
+{
+    if (!response_)
+        return;
+    static const json config_request_args { { "items", { { { "section", "hlasm" } } } } };
+    static uint64_t config_request_number = 0;
+    response_->request("config_request_" + std::to_string(config_request_number),
+        "workspace/configuration",
+        config_request_args,
+        std::bind(&feature_workspace_folders::configuration, this, std::placeholders::_1, std::placeholders::_2));
+    ++config_request_number;
+}
+
+void feature_workspace_folders::configuration(const json&, const json& params)
+{
+    if (params.size() == 0)
+        LOG_WARNING("Empty configuration response received.");
+
+    lib_config::load_from_json(params[0]);
```

Comment:
does this make sence when `params.size() == 0`?

Reply:
No, it does not, thanks.